app/templates/components/ui-table/cell/cell-roles.hbs

```diff
@@ -1,18 +1,18 @@
 <div class="ui ordered list">
-  <div class="item">{{this.record.email}} ({{t 'Owner'}})</div>
-  {{#each this.extraRecords.organizers as |role|}}
-    <div class="item">{{role.email}} ({{t 'Organizer'}})</div>
-  {{/each}}
-  {{#each this.extraRecords.coorganizers as |role|}}
-    <div class="item">{{role.email}} ({{t 'Co-organizers'}})</div>
-  {{/each}}
-  {{#each this.extraRecords.trackOrganizers as |role|}}
-    <div class="item">{{role.email}} ({{t 'Track-organizers'}})</div>
-  {{/each}}
-  {{#each this.extraRecords.registrars as |role|}}
-    <div class="item">{{role.email}} ({{t 'Registrars'}})</div>
-  {{/each}}
-  {{#each this.extraRecords.moderators as |role|}}
-    <div class="item">{{role.email}} ({{t 'Moderators'}})</div>
-  {{/each}}
-</div>
\ No newline at end of file
+  <div class="item word-break">{{this.record.email}} ({{t 'Owner'}})</div>
+    {{#each this.extraRecords.organizers as |role|}}
```

Comment:
still indentation is wrong. Pls fix this

Reply:
@daretobedifferent18 please tell me what is the correct indentation for this file , don't i have to indent "#each" statements

Reply:
just add word-break class and change nothing.